service/src/test/resources/application-test.yaml

```diff
@@ -14,7 +14,7 @@ tanagra:
     password: dbpwd
 
   vumc-admin:
-    base-path: https://tanagra-dev.api.verily.com
+    base-path: https://tanagra-test.app.vumc.org
     oauth-client-id: 694046000181-baqla782scqfaaga26869orhhd8o7vjo.apps.googleusercontent.com
```

Comment:
The URL and OAuth client id here both refer to a specific deployment of the admin service, so I would expect both to change together.

Given what you said before about not wanting to check the OAuth client id for the VUMC admin service deployment into the core public repo, I think it would make sense to leave these 2 properties as is (i.e. pointing to the Verily admin service deployment) and add a comment above how to set these properties when running locally (e.g. VUMC admin service URL is ... and you can get the OAuth client id from the console or talking to a VUMC data apps eng...?)